### PR TITLE
Consolidate port vmem tests.

### PR DESCRIPTION
**Pull request by dnakamura**
Merge PortVMemTest into PortVmemTest.

Signed-off-by: Devin Nakamura <devinn@ca.ibm.com>

### CODE REVIEW

File: fvtest/porttest/omrvmemTest.cpp [hunk 1]
```diff
@@ -4207,6 +4207,24 @@ TEST(PortVmemTest, vmem_testGetProcessMemorySize)
...
+ */
+TEST(PortVmemTest, omrvmem_get_available_physical_memory)
+{
+	OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
+	uint64_t freePhysicalMemorySize = 0;
+	int32_t status = omrvmem_get_available_physical_memory(&freePhysicalMemorySize);
+	/* omrvmem_get_available_physical_memory is not supported on zos */
+#if !defined(J9ZOS390)
+	ASSERT_EQ(0, status) << "Non-zero status from omrvmem_get_available_physical_memory";
+	portTestEnv->log("freePhysicalMemorySize = %llu\n", freePhysicalMemorySize);
```
Comment (rwy0717) [inline fvtest/porttest/omrvmemTest.cpp]: While you're here, the printf string should be `...%" OMR_PRIu64 "...`, defined by `"omrformatconsts.h"`

File: fvtest/porttest/omrvmemTest.cpp [hunk 2]
```diff
@@ -4207,6 +4207,24 @@ TEST(PortVmemTest, vmem_testGetProcessMemorySize)
...
+#else /* !defined(J9ZOS39) */
```
Comment (rwy0717) [inline fvtest/porttest/omrvmemTest.cpp]: typo: missing the zero in s390

Comment (rwy0717) [reply fvtest/porttest/omrvmemTest.cpp]: (missing below as well)

**Comment by charliegracie** [comment]
@genie-omr build all

**Comment by charliegracie** [comment]
@rwy0717 are you good with these changes now?

**Comment by rwy0717** [comment]
LGTM

**Pull request merged**